website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -442,9 +442,9 @@ To configure this inside of dbt Cloud, use the [extended attributes feature](/do
 
 compute:
   Compute1:
-    http_path:[`/some/other/path']
+    http_path: /SOME/OTHER/PATH
   Compute2:
-    http_path:[`/some/other/path']
+    http_path: /SOME/OTHER/PATH
 
 ```
 
```

Comment:
I noticed the profile.yml Under [Selecting compute per model](https://docs-getdbt-bwqcghv0s-dbt-labs.vercel.app/reference/resource-configs/databricks-configs#selecting-compute-per-model) also uses the `<profile-name>` but maybe we should change all profile.yml and project.yml examples in one PR to reduce confusion. what do you think?